sui/src/config/mod.rs

```diff
@@ -222,16 +249,31 @@ impl NetworkConfig {
                         .public_key
                         .make_narwhal_public_key()
                         .expect("Can't get narwhal public key");
+
+                    debug!("Resolving {}:{} for {}", &x.host, &x.port, name);
+
                     let primary = PrimaryAddresses {
-                        primary_to_primary: socket_addr_from_hostport(&x.host, x.port + 100),
-                        worker_to_primary: socket_addr_from_hostport(&x.host, x.port + 200),
+                        primary_to_primary: socket_addr_from_hostport_retry(&x.host, x.port + 100),
+                        worker_to_primary: socket_addr_from_hostport_retry(&x.host, x.port + 200),
                     };
+                    let p2w = socket_addr_from_hostport_retry(&x.host, x.port + 300);
+                    let w2w = socket_addr_from_hostport_retry(&x.host, x.port + 400);
+
+                    debug!(
+                        "p2p: {}, w2p: {}, p2w: {}, consensus: {}, w2w: {}",
```

Comment:
Again, please split every `{}` into a field.  We don't want the main log message to use any interpolation, this way we can easily parse the fields in log aggregators that support it.